package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "homebridge-deebot",
   "alias": "Deebot",
-  "version": "5.1.0",
+  "version": "5.1.0-support-custom-areas",
```

Comment:
if you can remove this change that would be super 👍

Reply:
This was meant just for me to be able to recognize the development version when installed. I thought you would just not merge it... I should probably have kept that local. Removed.